### DISCUSSION TITLE
node-gyp in renovate self-hosted

### DISCUSSION THREAD

**Discussion by benasher44** [renovatebot/renovate#13530] [Request Help] [answered]
Hi! We're using renovate self-hosted via the github-action. One of our dependencies uses node-gyp, which requires python3. How can I get renovate to have python available? From the logs, it looks like renovate is running in docker. Is there a way to get it to use python in the docker image? I can have python set up in the CI pipeline easily, but I don't think that helps, if it's running in docker.

Thanks in advance!

**Comment by viceice** [comment, marked as answer]
No, as normally renovate don't execute any scripts, even node-gyp scripts are blocked.

So please describe more in detail what you try to achive.

**Reply by benasher44** [reply]
Ah this makes sense. I temporarily enabled scripts to workaround a yarn bug, so that makes sense why this would have started being an issue for us all of a sudden. This is helpful. Thanks!

**Reply by benasher44** [reply]
@viceice we have a monorepo setup with many packages. I only need scripts enabled in one of those packages. Do you know if it's possible to set it up so that scripts are enabled only in that one sub-package? Thanks again for your help

**Reply by viceice** [reply]
nope, not possible.
